lib/irb.rb

```diff
@@ -744,6 +744,12 @@ def handle_exception(exc)
         puts message
       end
       print "Maybe IRB bug!\n" if irb_bug
+    rescue Exception
+      begin
+        print "#{exc.inspect}\n"
```

Comment:
I think we should hint users that something happened in the `handle_exception`, like:

```suggestion
    rescue Exception => handler_exc
      begin
        puts exc.inspect
        puts "backtraces are hidden because #{handler_exc} was raised when processing them"
```

Reply:
👍  Updated